schemas/package.yml

```
@@ -9,6 +9,12 @@
   type: group
   fields:
 
+    - name: namespace
+      level: extended
+      type: keyword
+      description: Package's namespace
```

Can we expand on the description a bit? I offered a suggestion here but feel free to adjust.

```suggestion
      short: Package's namespace
      description: > 
        Package's namespace.
        
        Some software registries and repositories allow for a namespace as part of a
        package's naming, such as Maven groupId and npm scopes.
        
```

---

@rgmz how do you feel about these suggestions from the last review?